plugin/src/makeWorklet.ts

```diff
@@ -323,52 +101,7 @@ export function makeWorklet(
   assert(transformed, "'transformed' is undefined");
   assert(transformed.ast, "'transformed.ast' is undefined");
 
-  traverse(transformed.ast, {
-    Identifier(path) {
-      if (!path.isReferencedIdentifier()) {
-        return;
-      }
-      const name = path.node.name;
-      // if the function is named and was added to globals we don't want to add it to closure
-      // hence we check if identifier has that name
-      if (
-        globals.has(name) ||
-        ('id' in fun.node && fun.node.id && fun.node.id.name === name)
-      ) {
-        return;
-      }
-
-      const parentNode = path.parent;
-
-      if (
-        isMemberExpression(parentNode) &&
-        parentNode.property === path.node &&
-        !parentNode.computed
-      ) {
-        return;
-      }
-
-      if (
-        isObjectProperty(parentNode) &&
-        isObjectExpression(path.parentPath.parent) &&
-        path.node !== parentNode.value
-      ) {
-        return;
-      }
-
-      let currentScope = path.scope;
-
-      while (currentScope != null) {
-        if (currentScope.bindings[name] != null) {
-          return;
-        }
-        currentScope = currentScope.parent;
-      }
-      closure.set(name, path.node);
-    },
-  });
-
-  const variables = Array.from(closure.values());
+  const variables = makeArrayFromCapturedBindings(transformed.ast, fun);
```

Comment:
What's the reason behind this rename from `closure`?

Reply:
Previously we were doing a closure inside the function and then we were doing the array from its values as in mentioned line. I moved it to another function which returns this array so the code could be more readable.